Actions/CreateReleaseNotes/README.md

```diff
@@ -14,6 +14,7 @@ none
 | :-- | :-: | :-- | :-- |
 | shell | | The shell (powershell or pwsh) in which the PowerShell script in this action should run | powershell |
 | token | | The GitHub token running the action | github.token |
+| appVersion | Yes | App version | |
```

Comment:
Minor: I'd call this simply _version_ or _releaseVersion_ as it's not necessarily related to the app version (there could even be apps with different versions in the same release, right?)

Reply:
It is called appVersion because that is the name in CreateRelease.yaml as well - build version would be better for both

Reply:
renamed to build version